### PR TITLE
integrate eslint with extends airbnb-base, write some rule and change…

### PR DESCRIPTION
**Pull request by muzammal-invozone**
… filename

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// var count = 1;
```
Comment (isheraz) [inline index.js]: Remove all commented code before commiting.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+// if (true) {
+//   count += 1;
+// }
+
+// good, use the let.
+let counts = 1;
```
Comment (isheraz) [inline index.js]: Can you explain what we're doing here???

File: test_lint.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+// var count = 1;
```
Comment (isheraz) [inline test_lint.js]: remove test files

File: package.json [hunk 4]
```diff
@@ -28,9 +28,13 @@
...
   "devDependencies": {
+    "eslint": "^7.16.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.1.0",
     "husky": "^4.3.6"
   },
-  "husky" : {
+  "husky": {
     "precommit": "echo 'working'"
```
Comment (isheraz) [inline package.json]: fix the pre-commit hook to console.log(); env variables

**Review by isheraz — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (muzammal-invozone) [reply index.js]: we used no-var airbnb rule, it is the true case that we are initializing a variable with let keyword.

**Review by isheraz — APPROVED** [review]

**Review by isheraz — APPROVED** [review]

**Pull request merged**